src/main/java/org/folio/fqm/service/QueryManagementService.java

```diff
@@ -193,6 +193,15 @@ public List<List<String>> getSortedIds(UUID queryId, int offset, int limit) {
   }
 
   public List<Map<String, Object>> getContents(UUID entityTypeId, List<String> fields, List<List<String>> ids) {
+    entityTypeService
+      .getEntityTypeDefinition(entityTypeId)
+      .map(IdColumnUtils::getIdColumnNames)
+      .orElseThrow(() -> new EntityTypeNotFoundException(entityTypeId))
+      .forEach(colName -> {
+        if (!fields.contains(colName)) {
+          fields.add(colName);
+        }
+      });
```

Comment:
We didn't previously need this, since we ensured that any query or list that was saved included id columns in its list of fields. But now that we're allowing a subset of columns to be selected (and in the future, only these columns will be sent from the UI to the BE), we need to make sure that the id columns are always included in the results, whether they're included in the request or not.